pages/account/manage/milestone/[[...data]].js

```diff
@@ -226,6 +246,15 @@ export default function ManageMilestone({ BASE_URL, milestone }) {
                     <p className="text-sm text-primary-low-medium">
                       For example: <i>DD / MM / YYYY</i>
                     </p>
+                    <Select
+                      name="layout"
+                      label="Select format"
+                      value={dateFormat}
+                      options={options}
+                      onChange={(e) => {
+                        setdateFormat(e.target.value);
+                      }}
```

Comment:
I think this can be a single line?

```suggestion
                      onChange={(e) => setdateFormat(e.target.value)}
```

Reply:
Resolved but I don't see a comment or changes?

Reply:
Oops sorry I accidentally discard it now it's done.